libs/owasp-diff-runner/src/main/java/ddf/lib/OwaspDiffRunner.java

```diff
@@ -53,6 +54,7 @@
 
   private static Invoker invoker = new DefaultInvoker();
 
+  @SuppressWarnings("squid:S3010" /* Constructor only here for associated unit test */)
```

Comment:
✏️  Then make it package private and annotate it with `@VisibleForTesting` instead of doing a suppression

Reply:
SonarQube has an issue with setting static variables in the constructor. After I made my changes I believe @coyotesqrl (Didn't mean to throw you under the bus) mentioned that it was used for testing and I should just suppress instead.